Sources/Logging.swift

```diff
@@ -6,13 +7,8 @@
 //  Copyright © 2017 flowkey. All rights reserved.
 //
 
-import SDL
-
-#if os(Android)
+@_implementationOnly import SDL
```

Comment:
I found some docs about this [here](https://github.com/apple/swift/blob/main/docs/ReferenceGuides/UnderscoredAttributes.md#_implementationonly) but can you explain in our context what this actually changes?